src/podcast/downloader.py

```
@@ -0,0 +1,585 @@
+            )
+
+            return DownloadResult(
+                episode_id=episode.id,
+                success=True,
+                local_path=output_path,
+                file_size=file_size,
+                file_hash=file_hash,
+                duration_seconds=duration,
+            )
+
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)